### ISSUE TITLE
Echo doesn't find any devices.

### ISSUE THREAD

**Issue by hughwilliams94** [bwssytems/ha-bridge#825]
Hi, I am very new to ha-bridge and I am just looking to trigger some local script on a ubuntu laptop running ha-bridge. I start the program by running the .jar file with this command ```sudo java -jar ha-bridge-5.0.0.jar``` and it runs fine, I can access the webpage at ```localhost``` and create a device which successfully triggers the script when tested. The problem is that try as I might, my echo dot does not find any devices on my home network. I have upnp enabled and there does appear to be some interaction in the trace (which I have attached) but not outcome. Any ideas? Sorry if the answer is simple but I honestly have tried looking through other issues and google and can't make head nor tails of the problem!

Attached trace:

```
 ✘ hugh@galliumos  ~/habridge  sudo java -jar ha-bridge-5.0.0.jar                                                             ~/habridge
2017-12-01 14:53:15,239 [main] INFO  com.bwssystems.HABridge.HABridge - HA Bridge (v5.0.0) starting....
2017-12-01 14:53:15,295 [main] INFO  com.bwssystems.HABridge.BridgeSettings - reading from config file: data/habridge.config
2017-12-01 14:53:15,776 [main] INFO  com.bwssystems.HABridge.HABridge - HA Bridge initializing....
2017-12-01 14:53:16,019 [main] INFO  spark.staticfiles.StaticFilesConfiguration - StaticResourceHandler configured with folder = /public
2017-12-01 14:53:16,061 [main] INFO  com.bwssystems.HABridge.SystemControl - System control service started....
2017-12-01 14:53:16.307:INFO::Thread-0: Logging initialized @1826ms to org.eclipse.jetty.util.log.StdErrLog
2017-12-01 14:53:16,343 [main] INFO  com.bwssystems.HABridge.util.UDPDatagramSender - Initializing UDP response Socket...
2017-12-01 14:53:16,351 [main] INFO  com.bwssystems.HABridge.util.UDPDatagramSender - UDP response Seocket initialized to: 50000
2017-12-01 14:53:16,357 [main] INFO  com.bwssystems.HABridge.plugins.harmony.HarmonyHome - Harmony Home created. No Harmony devices configured.
2017-12-01 14:53:16,362 [main] INFO  com.bwssystems.HABridge.plugins.NestBridge.NestHome - Nest Home created. No Nest configured.
2017-12-01 14:53:16,365 [main] INFO  com.bwssystems.HABridge.plugins.hue.HueHome - Hue passthru Home created. No Hue passtrhu systems configured.
2017-12-01 14:53:16,389 [main] INFO  com.bwssystems.HABridge.plugins.hal.HalHome - HAL Home created. No HAL devices configured.
2017-12-01 14:53:16,392 [main] INFO  com.bwssystems.HABridge.plugins.mqtt.MQTTHome - MQTT Home created. No MQTT Clients configured.
2017-12-01 14:53:16,410 [main] INFO  com.bwssystems.HABridge.plugins.hass.HassHome - HomeAssistant Home created. No HomeAssistants configured.
2017-12-01 14:53:16,413 [main] INFO  com.bwssystems.HABridge.plugins.exec.CommandHome - Command Home for system program execution created.
2017-12-01 14:53:16,419 [main] INFO  com.bwssystems.HABridge.plugins.http.HTTPHome - Http Home created.
2017-12-01 14:53:16,421 [main] INFO  com.bwssystems.HABridge.plugins.tcp.TCPHome - TCP Home created.
2017-12-01 14:53:16,426 [main] INFO  com.bwssystems.HABridge.plugins.udp.UDPHome - UDP Home created.
2017-12-01 14:53:16,428 [main] INFO  com.bwssystems.HABridge.plugins.vera.VeraHome - Vera Home created. No Veras configured.
2017-12-01 14:53:16,439 [main] INFO  com.bwssystems.HABridge.plugins.fibaro.FibaroHome - Fibaro Home created. No Fibaros configured.
2017-12-01 14:53:16,443 [main] INFO  com.bwssystems.HABridge.plugins.domoticz.DomoticzHome - Domoticz Home created. No Domoticz devices configured.
2017-12-01 14:53:16,446 [main] INFO  com.bwssystems.HABridge.plugins.somfy.SomfyHome - Somfy Home created. No Somfys configured.
2017-12-01 14:53:16,449 [Thread-0] INFO  spark.embeddedserver.jetty.EmbeddedJettyServer - == Spark has ignited ...
2017-12-01 14:53:16,454 [Thread-0] INFO  spark.embeddedserver.jetty.EmbeddedJettyServer - >> Listening on 0.0.0.0:80
2017-12-01 14:53:16,455 [main] INFO  com.bwssystems.HABridge.plugins.lifx.LifxHome - LifxDevice Home created. No LifxDevices configured.
2017-12-01 14:53:16.462:INFO:oejs.Server:Thread-0: jetty-9.4.z-SNAPSHOT
2017-12-01 14:53:16,506 [main] WARN  com.bwssystems.HABridge.dao.GroupRepository - Error reading the file: data/group.db - Does not exist or is not readable. continuing...
2017-12-01 14:53:16,507 [main] INFO  com.bwssystems.HABridge.devicemanagmeent.DeviceResource - HABridge device management service started.... 
2017-12-01 14:53:16,563 [main] INFO  com.bwssystems.HABridge.hue.HueMulator - Hue emulator service started....
2017-12-01 14:53:16.588:INFO:oejs.session:Thread-0: DefaultSessionIdManager workerName=node0
2017-12-01 14:53:16.588:INFO:oejs.session:Thread-0: No SessionScavenger set, using defaults
2017-12-01 14:53:16.610:INFO:oejs.session:Thread-0: Scavenging every 660000ms
2017-12-01 14:53:16.690:INFO:oejs.AbstractConnector:Thread-0: Started ServerConnector@54f83f6e{HTTP/1.1,[http/1.1]}{0.0.0.0:80}
2017-12-01 14:53:16.692:INFO:oejs.Server:Thread-0: Started @2216ms
2017-12-01 14:53:16,693 [main] INFO  com.bwssystems.HABridge.HABridge - Traceupnp: upnp config address: 192.168.1.87-useIface:false on web server: 0.0.0.0:80
2017-12-01 14:53:16,698 [main] INFO  com.bwssystems.HABridge.upnp.UpnpSettingsResource - Description xml service started....
2017-12-01 14:53:16,720 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener starting....
2017-12-01 14:53:16,723 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: Interface: wlp2s0 valid usable IP address: /192.168.1.87
2017-12-01 14:53:16,725 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: Adding wlp2s0 to our upnp join interface set.
2017-12-01 14:53:16,726 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: Interface: lo valid usable IP address: /127.0.0.1
2017-12-01 14:53:16,726 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: Adding lo to our upnp join interface set.
2017-12-01 14:53:16,726 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener running and ready....
2017-12-01 14:53:53,738 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: SSDP M-SEARCH packet from 192.168.1.84:50000
2017-12-01 14:53:53,742 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 1 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,743 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: upnp:rootdevice
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19::upnp:rootdevice

>>>
2017-12-01 14:53:53,746 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 2 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,747 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>
2017-12-01 14:53:53,749 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 3 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,749 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: urn:schemas-upnp-org:device:basic:1
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>
2017-12-01 14:53:53,749 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: SSDP M-SEARCH packet from 192.168.1.84:50000
2017-12-01 14:53:53,750 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 1 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,750 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: upnp:rootdevice
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19::upnp:rootdevice

>>>
2017-12-01 14:53:53,751 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 2 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,751 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>
2017-12-01 14:53:53,752 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Traceupnp: send upnp discovery template 3 with response address: 192.168.1.87:80 to address: /192.168.1.84:50000
2017-12-01 14:53:53,752 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - Sending response string: <<<HTTP/1.1 200 OK
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
EXT:
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
hue-bridgeid: 4C8093FFFEC97C19
ST: urn:schemas-upnp-org:device:basic:1
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>
2017-12-01 14:54:13,766 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - sendUpnpNotify notifyTemplate is <<<NOTIFY * HTTP/1.1
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
NTS: ssdp:alive
hue-bridgeid: 4C8093FFFEC97C19
NT: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>
2017-12-01 14:54:13,776 [main] DEBUG com.bwssystems.HABridge.upnp.UpnpListener - sendUpnpNotify notifyTemplate is <<<NOTIFY * HTTP/1.1
HOST: 239.255.255.250:1900
CACHE-CONTROL: max-age=100
LOCATION: http://192.168.1.87:80/description.xml
SERVER: Linux/3.14.0 UPnP/1.0 IpBridge/1.19.0
NTS: ssdp:alive
hue-bridgeid: 4C8093FFFEC97C19
NT: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19
USN: uuid:2f402f80-da50-11e1-9b23-4c8093c97c19

>>>


```

**Comment by hughwilliams94** [comment]
Just to note the machine running ha-bridge is at 197.168.1.87 and the dot is at 197.168.1.84

**Comment by Blacksus90** [comment]
It the new echo 2nd gen. It's been reported multiple times ([https://github.com/bwssytems/ha-bridge/issues/810](here) for example. No solution yet.

**Comment by bwssytems** [comment]
Unfortunately, this looks to be a second gen device that has known issues.

**Comment by bwssytems** [comment]
duplicate of #810

**Comment by bwssytems** [comment]
Please try the new release 5.1.0 https://github.com/bwssytems/ha-bridge/releases/tag/v5.1.0

**Comment by hughwilliams94** [comment]
Will try to this weekend thanks.

**Comment by bwssytems** [comment]
Any Update?

**Comment by hughwilliams94** [comment]
Sorry both my computers are out of action sorry. Will let you know if I can
get one of them back online.

**Comment by bwssytems** [comment]
Please go test RC version in #863

**Comment by Blacksus90** [comment]
Thank you so much for trying to fix this - habridge is the only reason Alexa is worth it to me.

#863 did not fix it for me though - my echo gen 2 can still not find any smart home devices.

**Comment by audiofreak9** [comment]
@Blacksus90 
There is a new setting called "UPNP Send Delay", try playing with this setting.  Default is 1500, try 1000, or 2000, etc...

**Comment by dmsims** [comment]
Also check the IP address in Habridge matches the computer (sorry but's been done :))

**Comment by Blacksus90** [comment]
The IP matches, no worries ;)
I have tried it in 250 increments for the delay between 100 and 15000 - no luck so far.

**Comment by bwssytems** [comment]
In the Amazon Echo App on your phone, do a add scene as it will go and discover again but it seems it may be different.

**Comment by myhaus** [comment]
Hey folks, just joined. So am I correct that at the moment HA-Bridge will not be discovered by the 2nd gen Echo at all? Or are some people having success with this?
Thanks

**Comment by kevvv31** [comment]
Hi,
Yes it looks like many people have the same issue.. I have 3 new Echo (middle size 2nd gen) and none of them are able to discover devices on HA Bridge (other bridges like Hue & Yonomi are correctly found).
I was very happy to use HA Bridge to add Smart Devices to Alexa without the need of asking a skill to do something.
I hope it will works again soon :)

**Comment by myhaus** [comment]
Might I suggest some kind of notification on the BWS homepage or bridge forum page that there is currently an issue with 2nd gen Echos. Would have saved me a lot of time and frustration.
Thanks

**Comment by bwssytems** [comment]
So V5.2.0 is out there and it has an update for upnp sending. Please try and see if your echo 2nd gen is working now.

**Comment by myhaus** [comment]
Not seeing any difference here. Still not being discovered, however, I only own a 2nd gen Echo and have never gotten the bridge to work so I may be doing something else wrong. Maybe someone that has 1st and 2nd gen Echos can confirm that it works with gen 1 but not with gen 2. Thanks again

**Comment by bwssytems** [comment]
My scenario that seems to work:

** Make sure you have enabled the 'Use Link Button' in the ha-bridge security settings!

start discover on Alexa
press the link button on the ha-bridge 'Bridge Devices' tab
most likely this discover will not find anything
start discover on Alexa again
press the link button on the ha-bridge 'Bridge Devices' tab
after Alexa finishes, you should see your devices or new devices added

**Comment by bwssytems** [comment]
Any updates?

**Comment by myhaus** [comment]
No luck, tried changing router and bridge settings, but again I may have any number of things set wrong. here is some log...

03-16-2018 17:43:09.868 | INFO | HA Bridge startup sequence... | com.bwssystems.HABridge.HABridge
-- | -- | -- | --
03-16-2018 17:43:09.911 | INFO | reading from config file: data/habridge.config | com.bwssystems.HABridge.BridgeSettings
03-16-2018 17:43:10.722 | INFO | HA Bridge (v5.2.0) initializing.... | com.bwssystems.HABridge.HABridge
03-16-2018 17:43:10.755 | INFO | StaticResourceHandler configured with folder = /public | spark.staticfiles.StaticFilesConfiguration
03-16-2018 17:43:10.776 | INFO | System control service started.... | com.bwssystems.HABridge.SystemControl
03-16-2018 17:43:10.843 | INFO | Initializing UDP response Socket... | com.bwssystems.HABridge.util.UDPDatagramSender
03-16-2018 17:43:10.851 | INFO | UDP response Socket initialized to: 50000 | com.bwssystems.HABridge.util.UDPDatagramSender
03-16-2018 17:43:10.857 | INFO | HTTP Home created. | com.bwssystems.HABridge.plugins.http.HTTPHome
03-16-2018 17:43:10.863 | INFO | Harmony Home created. No Harmony devices configured. | com.bwssystems.HABridge.plugins.harmony.HarmonyHome
03-16-2018 17:43:10.866 | INFO | Nest Home created. No Nest configured. | com.bwssystems.HABridge.plugins.NestBridge.NestHome
03-16-2018 17:43:10.868 | INFO | Hue passthru Home created. No Hue passtrhu systems configured. | com.bwssystems.HABridge.plugins.hue.HueHome
03-16-2018 17:43:10.869 | INFO | HAL Home created. No HAL devices configured. | com.bwssystems.HABridge.plugins.hal.HalHome
03-16-2018 17:43:10.871 | INFO | MQTT Home created. No MQTT Clients configured. | com.bwssystems.HABridge.plugins.mqtt.MQTTHome
03-16-2018 17:43:10.873 | INFO | HomeAssistant Home created. No HomeAssistants configured. | com.bwssystems.HABridge.plugins.hass.HassHome
03-16-2018 17:43:10.875 | INFO | HomeWizard Home created. No HomeWizard gateways configured. | com.bwssystems.HABridge.plugins.homewizard.HomeWizardHome
03-16-2018 17:43:10.876 | INFO | Command Home for system program execution created. | com.bwssystems.HABridge.plugins.exec.CommandHome
03-16-2018 17:43:10.878 | INFO | TCP Home created. | com.bwssystems.HABridge.plugins.tcp.TCPHome
03-16-2018 17:43:10.881 | INFO | UDP Home created. | com.bwssystems.HABridge.plugins.udp.UDPHome
03-16-2018 17:43:10.882 | INFO | Vera Home created. No Veras configured. | com.bwssystems.HABridge.plugins.vera.VeraHome
03-16-2018 17:43:10.883 | INFO | Fibaro Home created. No Fibaros configured. | com.bwssystems.HABridge.plugins.fibaro.FibaroHome
03-16-2018 17:43:10.886 | INFO | Domoticz Home created. No Domoticz devices configured. | com.bwssystems.HABridge.plugins.domoticz.DomoticzHome
03-16-2018 17:43:10.887 | INFO | Somfy Home created. No Somfys configured. | com.bwssystems.HABridge.plugins.somfy.SomfyHome
03-16-2018 17:43:10.890 | INFO | LifxDevice Home created. No LifxDevices configured. | com.bwssystems.HABridge.plugins.lifx.LifxHome
03-16-2018 17:43:10.892 | INFO | OpenHAB Home created. No OpenHABs configured. | com.bwssystems.HABridge.plugins.openhab.OpenHABHome
03-16-2018 17:43:10.894 | INFO | FHEM Home created. No FHEMs configured. | com.bwssystems.HABridge.plugins.fhem.FHEMHome
03-16-2018 17:43:10.899 | INFO | Broadlink Home created. No Broadlinks configured. | com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome
03-16-2018 17:43:10.919 | WARN | Error reading the file: data\group.db - Does not exist or is not readable. continuing... | com.bwssystems.HABridge.dao.GroupRepository
03-16-2018 17:43:10.920 | INFO | HABridge device management service started.... | com.bwssystems.HABridge.devicemanagmeent.DeviceResource
03-16-2018 17:43:10.923 | INFO | == Spark has ignited ... | spark.embeddedserver.jetty.EmbeddedJettyServer
03-16-2018 17:43:10.936 | INFO | >> Listening on 0.0.0.0:82 | spark.embeddedserver.jetty.EmbeddedJettyServer
03-16-2018 17:43:10.968 | INFO | Hue emulator service started.... | com.bwssystems.HABridge.hue.HueMulator
03-16-2018 17:43:11.141 | INFO | Traceupnp: upnp config address: 192.168.10.125-useIface:false on web server: 0.0.0.0:82 | com.bwssystems.HABridge.HABridge
03-16-2018 17:43:11.142 | INFO | Description xml service started.... | com.bwssystems.HABridge.upnp.UpnpSettingsResource
03-16-2018 17:43:11.173 | INFO | UPNP Discovery Listener starting.... | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:43:11.256 | INFO | Traceupnp: Interface: lo valid usable IP address: /127.0.0.1 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:43:11.257 | INFO | Traceupnp: Adding lo to our upnp join interface set. | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:43:11.257 | INFO | Traceupnp: Interface: eth0 valid usable IP address: /192.168.10.125 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:43:11.257 | INFO | Traceupnp: Adding eth0 to our upnp join interface set. | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:43:11.257 | INFO | UPNP Discovery Listener running and ready.... | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:15.883 | INFO | Traceupnp: hue api user create requested for device type: test_ha_bridge#192.168.10.125 and username: 493aa126dd964191a55b8763965f15f9 | com.bwssystems.HABridge.hue.HueMulator
03-16-2018 17:44:53.505 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:54.774 | INFO | Link button pressed.... | com.bwssystems.HABridge.SystemControl
03-16-2018 17:44:55.006 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:55.038 | INFO | Traceupnp: request of description.xml from: 192.168.10.127:82 filled in with address: 192.168.10.125:82 | com.bwssystems.HABridge.upnp.UpnpSettingsResource
03-16-2018 17:44:56.507 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:58.008 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:58.008 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:44:59.509 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:01.009 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:02.510 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:02.511 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:04.012 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:05.513 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:07.013 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:07.014 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:08.514 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:10.016 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:11.516 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:11.517 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:13.017 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:14.518 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:16.018 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:16.019 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:17.519 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:19.020 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:20.520 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:24.775 | INFO | Link button time ended.... | com.bwssystems.HABridge.LinkButtonPressed
03-16-2018 17:45:30.751 | INFO | Link button pressed.... | com.bwssystems.HABridge.SystemControl
03-16-2018 17:45:30.918 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:32.418 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:32.441 | INFO | Traceupnp: request of description.xml from: 192.168.10.127:82 filled in with address: 192.168.10.125:82 | com.bwssystems.HABridge.upnp.UpnpSettingsResource
03-16-2018 17:45:33.920 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:35.420 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:35.421 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:36.921 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:38.422 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:39.923 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:39.923 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:41.424 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:42.925 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:44.425 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:44.426 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:45.927 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:47.428 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:48.929 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:48.930 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:50.430 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:51.930 | INFO | Traceupnp: send upnp discovery template 2 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:53.431 | INFO | Traceupnp: send upnp discovery template 3 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:53.432 | INFO | Traceupnp: SSDP M-SEARCH packet from 192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener
03-16-2018 17:45:54.932 | INFO | Traceupnp: send upnp discovery template 1 with response address: 192.168.10.125:82 to address: /192.168.10.127:50000 | com.bwssystems.HABridge.upnp.UpnpListener

**Comment by bwssytems** [comment]
@myhaus I would suspect that you need to run on port 80. You have port 82.

**Comment by myhaus** [comment]
OMG it works. Thanks for your help. So it looks like the new HA-Bridge version resolved the gen2 issue. I had some conflict with port 80 and could never get the bridge to work with it. I was connecting through Alex10 and would get Java and .net errors. I uninstalled some programs and messed with a bunch of settings and I did the classic, change 5 things at once so I can't know what resolved the conflict.Thanks Again!
